pyproject.toml

```diff
@@ -41,7 +41,7 @@ doc = [
 home = "https://github.com/pyansys/pyansys-tools-report"
```

Comment:
These links will be shown in the PyPI page of the package:

```suggestion
Home = "https://reportdocs.pyansys.com/"
Source = "https://github.com/pyansys/pyansys-tools-report"
Tracker = "https://github.com/ansys/pyansys-tools-report/issues"
```

Reply:
Thanks! Adding to PR :)